src/EnergyPlus/Furnaces.cc

```diff
@@ -9794,7 +9869,7 @@ namespace Furnaces {
             FirstHVACIteration = false;
         }
         FanOpMode = int(Par(3));
-        CompOp = int(Par(4));
+        CompOp = static_cast<CompressorOperation>(Par(4));
```

Comment:
The `par(*)` values that are parameters and results of the residual functions had to be `static_cast`.

Reply:
These things need to be converted to lambdas anyway. There is already an open [issue](https://github.com/NREL/EnergyPlus/issues/9110) about that. That conversion will make these casts unnecessary.